src/Grid/Col/index.js

```diff
@@ -1,11 +1,31 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-export default function Col({ children, xs, sm, md, lg, offset, style }) {
+export default function Col({
+  children,
+  lg,
+  md = lg,
+  sm = md || lg,
+  xs = sm || md || lg,
+  offsetLg,
+  offsetMd = offsetLg,
+  offsetSm = offsetLg || offsetMd,
+  offsetXs = offsetLg || offsetMd || offsetSm,
+  style
```

Comment:
this looks strange. I know. 
What would this do is 
*  md = lg ===  set the md = lg when md is not parsed as property
*  sm = md || lg === to set the sm = md when sm is not parsed and to lg when not even md is parsed 
etc etc

Reply:
Erm...not sure what that means? Haha

Reply:
haha I will try to rephrase.
So the point is that when someone is having a column 
```
<Col lg=10> some things here </Col>
```
then in my understanding is that, it would be nice when `md=10` and `sm=10` and `xs=10` without one having to explicitly define all of those every time a Col is being defined.
A value should fallback to the previous defined value.

So this way when someone has define `lg=10` (for instance) all the other properties will fallback to the value `10`

Does how this makes more sense ?

Reply:
I see, is there a more intuitive way we can do this in the code? Maybe inside the render itself - I get what you're going for but I think a refactor is in order to make the idea more clear.